docs/source/reference/config.rst

```diff
@@ -286,6 +286,10 @@ Available fields and semantics:
             hostPath:
               path: /tmp
               type: Directory
+          - name: dshm          # Use this to modify the /dev/shm volume mounted by SkyPilot
+            emptyDir:
+              medium: Memory
+              sizeLimit: 3Gi    # Set a size limit for the /dev/shm volume
```

Comment:
Is `/dev/shm` a convention name that people would know? Just wondering if we should elaborate a bit for what is this for.

Reply:
`/dev/shm` is generally well understood as a shared memory, and explaining it here might add more clutter. It's usage depends on the application (e.g., Ray stores GCS state there, [pytorch uses it for IPC](https://github.com/pytorch/pytorch#docker-image)), so it's hard to generalize. Folks running docker containers would usually be familiar with this, so might be fine to leave it be?